src/pages/MergeReadiness/index.tsx

```diff
@@ -0,0 +1,410 @@
+            <FormattedMessage defaultMessage="Fee recipient" />
+          </Heading>
+          <Text className="mt20">
+            <FormattedMessage defaultMessage="Now that transactions must be processed by validators, the validators that propose blocks including these transactions are eligible to receive the transaction fee tips. These are also known as priority fees, and are the unburnt portion of gas fees." />
+          </Text>
+          <Text className="mt20">
+            <FormattedMessage defaultMessage="These fees are paid by whoever submitted the transaction and come in the form of ETH on the execution layer (Mainnet). These rewards are not accounted for in your validator balance which is maintained on the consensus layer." />
+          </Text>
+          <Alert variant="warning" className="my40">
+            <Text>
+              <FormattedMessage
+                defaultMessage="As such, stakers must provide a {feeRecipient} address to their consensus client in order to receive these rewards. This is a normal Ethereum address that you're used to."
+                values={{
+                  feeRecipient: (
+                    <strong>
+                      <FormattedMessage defaultMessage="fee recipient" />
+                    </strong>
+                  ),
+                }}
+              />
+            </Text>
+            <Text className="mt20">
+              <FormattedMessage defaultMessage="If you do not provide an address to your client, you will not receive transaction fees when your validator proposes blocks." />
+            </Text>
+          </Alert>
+          <Text className="mt20">
+            <FormattedMessage defaultMessage="See you consensus layer client documentation for client-specific instructions on how to set this." />
+          </Text>
+          <ul className="sub-checklist-item">
+            <li className="py5">
+              <Link
+                primary
+                inline
+                to="https://docs.prylabs.network/docs/execution-node/fee-recipient/"
+              >
+                Prysm: Configuring a Fee Recipient Address
+              </Link>
+            </li>
+            <li className="py5">
+              <Link
+                primary
+                inline
+                to="https://docs.teku.consensys.net/en/latest/HowTo/Prepare-for-The-Merge/#configure-the-fee-recipient"
+              >
+                Teku: Configure the fee recipient
+              </Link>
+            </li>
+            <li className="py5">
+              <Link
+                primary
+                inline
+                to="https://lighthouse-book.sigmaprime.io/suggested-fee-recipient.html"
+              >
+                Lighthouse: Suggested Fee Recipient
+              </Link>
+            </li>
+            <li className="py5">
+              <Link primary inline to="https://nimbus.guide/">
+                Nimbus: The Nimbus book
+              </Link>
```

Comment:
Had a had time finding a specific reference for setting the fee recipient on Nimbus. I linked out to their main guide, but I was unable to find information on this within the guide, so open to suggestions from others on a better link here.